apps/emqx/test/emqx_persistent_messages_SUITE.erl

```diff
@@ -379,6 +390,65 @@ t_publish_empty_topic_levels(_Config) ->
         emqtt:stop(Pub)
     end.
 
+t_message_gc_too_young(_Config) ->
+    %% Check that GC doesn't attempt to create a new generation if there are fresh enough
+    %% generations around.
+    ?check_trace(
+        ok = emqx_persistent_message_ds_gc_worker:gc(),
```

Comment:
Nit: It looks like this test relies on the fact that no more one retention interval (1s) passes since the beginning of the test and implicit initialization of the generations by the testcase. This could be a potential source of flakiness.

Reply:
This test doesn't customize the retention period, so it relies on the default value, 1 hour, being long enough for this test.

Reply:
I see, thanks. Maybe add a comment about that to prevent accidental breakage of the test by modifying global config in `init_per_(testcase|suite|group)`.